### DISCUSSION TITLE
Erklär-Clip zum SAE

### DISCUSSION THREAD

**Discussion by camueller** [camueller/SmartApplianceEnabler#391] [Allgemein]
Wenn man im Internet etwas sucht und deswegen eine vermeintlilche passende Seite ansurft, möchte man schnell erkennen können, ob diese Seite potentiell interessant ist. Wenn man sich dazu erst viel Text durchlesen muss, ist das eher frustierend.

Deshalb habe ich eine Erklär-Animation erstellt und auf der Homepage eingebunden:
https://github.com/camueller/SmartApplianceEnabler/blob/master/pics/clip/SaeClip.gif

Ist diese aus Eurer Sicht dazu geeignet zu visualisieren, was der SAE macht?

**Comment by sigma415** [comment]
Gibt es den Clip nicht mehr ? ich kriege mit dem Link nur Error 404 ...

**Comment by camueller** [comment]
Der Clip ist doch auf der Homepage eingebunden.

**Comment by andi5783** [comment]
Mach ich irgendwas falsch, oder ist das ohne Ton?

**Comment by camueller** [comment]
Nein, es ist eine Animation ohne Ton.

**Comment by andi5783** [comment]
Ah sorry, dein Post ist ja von 2022.  hatte den Clip mit dem Video verwechselt. Ich würde sagen der Clip passt 👍🏼
